File: app/src/main/java/vergecurrency/vergewallet/viewmodel/PassphraseVerificationViewModel.java
package vergecurrency.vergewallet.viewmodel;

import androidx.lifecycle.ViewModel;
import vergecurrency.vergewallet.service.model.PreferencesManager;


public class PassphraseVerificationViewModel extends ViewModel {

	public void setFirstLaunch(boolean isFirstLaunch) {
		PreferencesManager.setFirstLaunch(isFirstLaunch);
	}

	public void setPassphrase(String passphrase) {PreferencesManager.setPassphrase(passphrase);}
}

File: app/src/main/kotlin/vergecurrency/vergewallet/viewmodel/PassphraseVerificationViewModel.kt
package vergecurrency.vergewallet.viewmodel

import androidx.lifecycle.ViewModel
import vergecurrency.vergewallet.service.model.EncryptedPreferencesManager


class PassphraseVerificationViewModel : ViewModel() {

    /*fun setFirstLaunch(isFirstLaunch: Boolean) {
        PreferencesManager.isFirstLaunch = isFirstLaunch
    }*/

    fun setPassphrase(passphrase: ByteArray) {
        EncryptedPreferencesManager.passphrase = passphrase
    }
}
